examples/add_pedestal_noise_template.xml

```diff
@@ -0,0 +1,130 @@
+<container>
+    <!-- This xml takes a given mc set and adds the choosen data noise onto the event.
+         The Creator will be changed from Ceres -> Ceres_Pedestal
+        @parameters SET_noiseDB    The noise database jsonl file created from the fact-eventlist.
+        @parameters SET_dataFolder The folder containing the fact files, structured in the fact data folder convention.
+        @parameters input          The multidata json file (see processor). Containing the MC data which should be added.
+        @output     output         The name of the finished fits file.
+    -->
+    <properties url="classpath:/default/settings.properties" />
+
+    <service id="calibService" class="fact.calibrationservice.ConstantCalibService" />
+    
+    
+    <property name="noiseDB" value="file:${SET_noiseDB}" />
```

Comment:
Why SET? Why inconsistent use of capitalisation, snake and camel case?

Reply:
In this case it is just written like this, because this value needs to be set explicitly beforehand as erna only sets input, output and aux_source. It basically tells the user to SET the `noiseDB` parameter. Which he can do manually by hand or more automatically through a text processor.

Reply:
I don't get it. So we have a property that get's set to the value of `SET_` plus that name. Why? Why not directlty set the property?

Reply:
If that's the case, just leave it empty and do not imply there is a `SET_<name>` property.

Reply:
@tarrox could we easily create a noiseDB file for the events in the test data in the rescources? Then one could add this to the repo and also include it in the tests.